src/models/scatter.js

```diff
@@ -10,6 +10,7 @@ nv.models.scatter = function() {
         , width        = null
         , height       = null
         , color        = nv.utils.defaultColor() // chooses color
+        , borderColor  = null
```

Comment:
This option is not descriptive of what it does, it makes it sound like the whole chart border color.  Other options on the points are prefixed with point, so this should probably be something like "pointBorderColor"

You'll also want to add a description and example for this new option into documentation.html in the big json object at the bottom, so that it shows up on the docs page.